llvm/utils/TableGen/DecoderEmitter.cpp

```diff
@@ -982,11 +979,10 @@ void DecoderEmitter::emitTable(formatted_raw_ostream &OS, DecoderTable &Table,
 }
 
 void DecoderEmitter::emitInstrLenTable(formatted_raw_ostream &OS,
-                                       std::vector<unsigned> &InstrLen) const {
+                                       ArrayRef<unsigned> InstrLen) const {
   OS << "static const uint8_t InstrLenTable[] = {\n";
-  for (unsigned &Len : InstrLen) {
+  for (const unsigned &Len : InstrLen)
```

Comment:
Drop the reference. `unsigned` is cheap to copy.

Reply:
Done.